Arista/ConfigFiles/telegraf-ninja.conf

```diff
@@ -0,0 +1,30 @@
+# Configuration for influxdb server to send metrics to
+[[outputs.influxdb]]
+  ## The full HTTP or UDP endpoint URL for your InfluxDB instance.
+  ## Multiple urls can be specified as part of the same cluster,
+  ## this means that only ONE of the urls will be written to each interval.
+  # urls = ["udp://localhost:8089"] # UDP endpoint example
+  urls = ["http://planck:8086"] # required
+  ## The target database for metrics (telegraf will create it if not exists).
+  database = "qubit" # required
+  ## Retention policy to write to.
+  retention_policy = ""
+  ## Precision of writes, valid values are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  ## note: using "s" precision greatly improves InfluxDB compression.
+  precision = "s"
+
+  ## Write timeout (for the InfluxDB client), formatted as a string.
+  ## If not provided, will default to 5s. 0s means no timeout (not recommended).
+  timeout = "10s"
+
+  namepass = ["exec", "ninja*"]
```

Comment:
Will this script output "exec" measurement as well?

If not, then it's not required to add it in namepass.